pulser-simulation/pulser_simulation/simconfig.py

```diff
@@ -86,6 +92,13 @@ class SimConfig:
     epsilon: float = 0.01
     epsilon_prime: float = 0.05
     dephasing_prob: float = 0.05
+    depolarizing_prob: float = 0.05
+    gen_noise_probs: list[float] = field(
+        init=True, default_factory=list, repr=False
+    )
```

Comment:
It's because of the approximation we use for many bodies. 
All effective channels are not probability-wise as the amplitude damping and the phase damping channel.

Reply:
Our noise model is based on the fact that only one bit string is modified at the time, and it is only justified when "rates" of collapse operators are low.

Reply:
This is why I wanted to seperate collapse operators from their rates to make sure the approx is verified. Normally I must not allow any set of operators where all rates aren't all low. It will not give realistic result. Or as done for the dephasing and depolarizing channel, warn the user that the effective channel he proposed will not yield realistic results since all rates are not low enough.

Reply:
And as it's done for the dephasing and depolarizing channels, I warn the user whenever the rate of an operator is not low enough that it would yield unrealistic results according to the approx we used.

Reply:
How can I?

Reply:
Well, the probability associated with each operator (if the operators already include the probablity, as I'm suggesting) will simply be `op.dag() * op` right?

Reply:
If the probability is incorporated in the operator, it wouldn't be just `sigma_x`, it would be `op = sqrt(p)*sigma_x`

Reply:
So it is hard to extract p. In the case of pauli matrices it's simple but for a general Kraus Operator it isn't possible. The only relation we know about them is that `sum(op_k) = id`

Reply:
Yes, I think you are right. I was assuming the operator was a unitary, but this is not a requirement. Nevermind my suggestion then.